src/main.js

```diff
@@ -571,17 +580,51 @@ function TimekitBooking() {
   // Initilization method
   var init = function(suppliedConfig) {
 
-    // Handle config and defaults
-    setConfig(suppliedConfig);
+    // Start from local config
+    if (!suppliedConfig.widgetId && !suppliedConfig.widgetSlug) {
+      return start(suppliedConfig)
+    }
 
-    // Include library styles if enabled
-    if (config.includeStyles) {
-      includeStyles();
+    // Load remote config
+    return loadRemoteConfig(suppliedConfig)
+    .then(function (response) {
+      var mergedConfig = $.extend(true, {}, response.data.config, suppliedConfig);
+      start(mergedConfig)
+    })
+
+  };
+
+  // Load config from remote (embed or hosted)
+  var loadRemoteConfig = function(suppliedConfig) {
+
+    config = setConfigDefaults(suppliedConfig)
+    timekitSetupConfig();
+    if (suppliedConfig.widgetId) {
+      return timekit
+      .getEmbedWidget({ id: suppliedConfig.widgetId })
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["response" is defined but never used](https://www.codacy.com/app/laander/booking-js/file/2598583893/issues/source?bid=2969108&fileBranchId=3284650#l605)